Exercise 5.2 Create DropDown Menus/js/navigation.js

```diff
@@ -0,0 +1,43 @@
+class DropDownMenu {
+  constructor(options) {
+    this.mainMenu = $(options.mainMenuSelector);
+    this.menuItems = this.mainMenu.children(options.menuItemSelector);
+  }
+
+  init() {
+    this.menuItems.hover(this.hoverEventHandler.bind(this));
+  }
+
+  hoverEventHandler(event) {
+    let target = $(event.target);
+    let subMenu = target.children("ul");
+    if(event.type === "mouseenter") {
```

Comment:
No need to check for mouseenter type. Please go through with the functionality of $.fn.hover.